pallets/moonbeam-foreign-assets/src/evm.rs

```diff
@@ -40,6 +40,8 @@ const ERC20_PAUSE_GAS_LIMIT: u64 = 160_000; // highest failure: 150_500
 pub(crate) const ERC20_TRANSFER_GAS_LIMIT: u64 = 160_000; // highest failure: 154_000
 pub(crate) const ERC20_APPROVE_GAS_LIMIT: u64 = 160_000; // highest failure: 153_000
 const ERC20_UNPAUSE_GAS_LIMIT: u64 = 160_000; // highest failure: 149_500
+pub(crate) const ERC20_TOTAL_SUPPLY_GAS_LIMIT: u64 = 160_000; // highest failure: 150_500
+pub(crate) const ERC20_BALANCE_OF_GAS_LIMIT: u64 = 155_000; // Calculated effective gas: max(used: 24276, pov: 150736, storage: 0) = 150736
```

Comment:
`ERC20_BALANCE_OF_GAS_LIMIT` will never be less costly than `ERC20_TOTAL_SUPPLY_GAS_LIMIT`. Although this is not failing, lets set both constants to `160_000`.